server/session/session.go

```diff
@@ -538,3 +544,14 @@ func (s *Session) sendAvailableEntities(w *world.World) {
 	}
 	s.writePacket(&packet.AvailableActorIdentifiers{SerialisedEntityIdentifiers: serializedEntityData})
 }
+
+// SetServerSettings Sets the server settings data.
+// This will be used when a ServerSettingsRequest packet is
+// sent by the client.
+func (s *Session) SetServerSettings(form form.Form) {
```

Comment:
The documentation here is poor. You should look at the Go documentation style guide: https://go.dev/blog/godoc